# For the purpose of the 3.1 release, instead of sharing the binary and ooxml
# export, we introduce a separate sc/source/filter/xlsx that creates libxlsx -
# a filter for the XLSX files.  After the release, the changes will be
# reverted so that the binary and ooxml are shared again.
# The changes include:
#   sc/source/filter/eppt/* -> sc/source/filter/pptx/pptx-*
diff -rup sd/source/filter/eppt-orig/epptooxml.cxx sd/source/filter/eppt/epptooxml.cxx
--- sd/source/filter/pptx/pptx-epptooxml.cxx	2009-04-23 12:18:28.000000000 +0200
+++ sd/source/filter/pptx/pptx-epptooxml.cxx	2009-04-23 12:19:57.000000000 +0200
@@ -1124,7 +1124,8 @@ void PowerPointExport::ImplWriteSlide( s
     pFS->endElementNS( XML_p, XML_cSld );
 
     WriteTransition( pFS );
-    WriteAnimations( pFS );
+    // disable these until there's time to iron it out
+    // WriteAnimations( pFS );
 
     pFS->endElementNS( XML_p, XML_sld );
 
Only in sd/source/filter/eppt: epptooxml.cxx~
